src/robots/gazebo/uav.cc

```diff
@@ -37,12 +37,12 @@ void UAV::setPitch(float pitch)
     m_Pitch = pitch;
     sendCommand();
 }
-void UAV::setRoll(float roll)
+void UAV::setRoll(float right)
```

Comment:
what is this doing!?

Reply:
That's not an automated fix, though ``clang-tidy`` did flag it up. It has a warning that complains if the var names for the method declaration don't match those in the definition. It does sound a bit pedantic, but it can stop you doing dumb things like:
```c++
void rotate(size_t pixelsToLeft);
// ...
void rotate(size_t pixelsToRight)
{
    // ...
}
```

Reply:
I think that's a reasonable warning but maybe, in this case, either you or it picked the wrong one 😄

Reply:
Haha... Hmm yeah. I'll sort that

Reply:
Done! Are we good to merge now?

Reply:
All looks good apart from that one whitespace comment...